src/components/FolderCard.js

```diff
@@ -1,30 +1,64 @@
+import { useState, useEffect } from 'react';
+import { getFolders } from '../api';
 import { getTimeAgo } from '../getTimeAgo';
+import FloatingAddBtn from './FloatingAddBtn';
+import './FloatingAddBtn.css';
+import './FolderCard.css';
 
 function formatDate(value) {
   const date = new Date(value);
   return `${date.getFullYear()}. ${date.getMonth() + 1}. ${date.getDate()}`;
 }
 
-function FolderCard({ links }) {
-  if (!links) {
-    return <div>Loading...</div>;
-  }
+function FolderCard({ selectedTab }) {

```

Comment:
이 컴포넌트의 역할이 단순한 UI가 아니니까 Card 나 Tab 보다는 (시각적인 형태보다 데이터중심으로) Folder 목록으로 보는게 좋겠습니다.

```suggestion
function Folders({ selectedFolder }) { // or FolderList
```

<img width="907" alt="image" src="https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/42806386/5f7d63b5-1e6a-4c5e-8b47-cd7938eb2857">